kview/src/main/java/dev/ikm/komet/kview/mvvm/model/DataModelHelper.java

```diff
@@ -77,4 +104,90 @@ public static Set<ConceptEntity> fetchDescendentsOfConcept(ViewProperties viewPr
                 .collect(Collectors.toSet());
         return allDecendents;
     }
+
+    public static List<PatternEntityVersion> getMembershipPatterns() {
+        List<PatternEntityVersion> membershipPatternList = new ArrayList<>();
+        PrimitiveData.get().forEachPatternNid((patternNid) -> {
+            PatternEntityVersion patternEntityVersion = (PatternEntityVersion)
+                    Calculators.View.Default().stampCalculator().latest(patternNid).get();
```

Comment:
Should we pass in the view calculator instead of using the default (`Default()`)?

Reply:
maybe... this is the code Ryan gave us

Reply:
I'll give it a try

Reply:
this breaks... I have to revert